scripts/wsrep_sst_clone.sh

```diff
@@ -242,14 +244,19 @@ cleanup_donor()
     wsrep_log_info "Cleanup DONOR DONE."
 }
 
-cleanup_joiner()
+sig_cleanup_joiner()
 {
-    wsrep_log_info "Joiner cleanup. SST daemon PID: $CLONE_REAL_PID"
+    if [ -n "$CLONE_INSTANCE_PID" ]; then
+        kill -SIGKILL $CLONE_INSTANCE_PID /dev/null 2>&1 ||:
+    fi
+
+    if [ -n "$NC_PID" ]; then
+        kill -SIGKILL $NC_PID /dev/null 2>&1 ||:
```

Comment:
I know I am redundant but can we have a log message (in debug) that we are killing things (pid)